ESRGAN.ipynb

```diff
@@ -152,6 +152,18 @@
         "  model_list.append('4x_DigiManga.pth')\n",
         "#@markdown <font size=2>author: <i>EULA</i><br>scale: `4x`<br>models: `4x_DigiManga.pth`<br>info: <i>Upscale B/W Manga Images to 4x with great screentone effect</i></font>\n",
         "\n",
+        "4x-AnimeSharp = True #@param {type: \"boolean\"}\n",
+		"if 4x-AnimeSharp:\n",
+		"  m.download_url('https://mega.nz/file/CFZGHb4A#yo2NaVANd9-gk9n4lxA9zkzKPqVukZYpzendRVCinDw')\n",
+		"  model_list.append(' 4x-AnimeSharp.pth ')\n",
+		"#@markdown <font size=2>author: <i>kim0</i><br>scale: `4x`<br>models: `4x-AnimeSharp.pth`<br>info: <i>Upscale colored mangas with a nice x4 upscaling</i></font>\n",
+        "\n",
+		"4x-AnimeSharp-lite = True #@param {type: \"boolean\"}\n",
```

Comment:
Make this `False`
No need to pre-select the Lite model if full version is already selected.